aws/resource_aws_route53_resolver_rule_test.go

```diff
@@ -111,6 +111,64 @@ func TestAccAwsRoute53ResolverRule_basic(t *testing.T) {
 	})
 }
 
+func TestAccAwsRoute53ResolverRule_justDotDomainName(t *testing.T) {
+	var rule route53resolver.ResolverRule
+	resourceName := "aws_route53_resolver_rule.example"
```

Comment:
nit: use `.test`?

Reply:
I thought about that but didn't want to change all the existing configurations that use `example`.
This tests need some refactoring so could all be cleaned up in one pass.